components/bybit/actions/linear-order-cancel-all/linear-order-cancel-all.mjs

```diff
@@ -0,0 +1,31 @@
+import bybit from "../../bybit.app.mjs";
+
+export default {
+  name: "ByBit Linear Order Cancel All",
+  version: "0.0.1",
+  key: "bybit-linear-order-cancel-all",
+  description: "Cancel All Active Orders." +
+      " [reference](https://bybit-exchange.github.io/docs/futuresV2/linear/#t-cancelallactive)",
+  props: {
+    bybit,
+    symbol: {
+      propDefinition: [
+        bybit,
+        "symbol",
+      ],
+    },
+  },
+  type: "action",
+  methods: {},
```

Comment:
You can remove empty `methods` from other files too.

```suggestion
```

Reply:
Done